tests/Integration/Behaviour/Features/Context/Domain/TaxRulesGroupFeatureContext.php

```diff
@@ -79,4 +92,190 @@ public function assertLastErrorIsTaxRulesGroupNotFound(): void
     {
         $this->assertLastErrorIs(TaxRulesGroupNotFoundException::class);
     }
+
+    /**
+     * @When I add new tax rules group :taxRulesGroupReference with following properties:
+     */
+    public function createTaxRulesGroup(string $taxRulesGroupReference, TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+
+        $command = new AddTaxRulesGroupCommand(
+            $data['name'],
+            PrimitiveUtils::castStringBooleanIntoBoolean($data['is_enabled'])
+        );
+
+        /** @var TaxRulesGroupId $taxRulesGroupId */
+        $taxRulesGroupId = $this->getCommandBus()->handle($command);
+
+        SharedStorage::getStorage()->set($taxRulesGroupReference, $taxRulesGroupId->getValue());
+    }
+
+    /**
+     * @When I edit tax rules group :taxRulesGroupReference with following properties:
+     */
+    public function editTaxRulesGroup(string $taxRulesGroupReference, TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+
+        $taxRulesGroupId = SharedStorage::getStorage()->get($taxRulesGroupReference);
+        $command = new EditTaxRulesGroupCommand($taxRulesGroupId);
+        if (isset($data['name'])) {
+            $command->setName($data['name']);
+        }
+        if (isset($data['is_enabled'])) {
+            $command->setEnabled(PrimitiveUtils::castStringBooleanIntoBoolean($data['is_enabled']));
+        }
+        $this->getCommandBus()->handle($command);
+    }
+
+    /**
+     * @When /^I (enable|disable)? tax rules group "(.*)"$/
+     */
+    public function toggleStatusTaxRulesGroup(string $action, string $taxRulesGroupReference): void
```

Comment:
Add `StringToBoolTransformContext` into suite in behat.yml and it will automatically handle the type transformation.
Then your $action (i would also prefer renaming it to $status or $enable) can be typed as bool.
```suggestion
    public function toggleStatusTaxRulesGroup(bool $enable, string $taxRulesGroupReference): void
```

Reply:
Fixed